pyproject.toml

```diff
@@ -0,0 +1,94 @@
+[tool.poetry]
+name = "excludarr"
+version = "1.1.2"
+description = "Excludarr excludes streaming services from Radarr and Sonarr"
+authors = ["Haije Ploeg <ploeg.haije@gmail.com>", "barsa2000 <>"]
+license = "MIT"
+readme = "README.md"
+homepage = "https://github.com/excludarr/excludarr"
+repository = "https://github.com/excludarr/excludarr"
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Topic :: Software Development :: Build Tools",
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3 :: Only",
+]
+keywords = [
+    "excludarr",
+    "radarr",
+    "sonarr",
+    "diskspace",
+    "cleanup",
+    "management",
+    "library",
+    "streaming",
+    "netflix",
+]
+include = [
+    { path = ".examples/", format = ["sdist"] },
+    { path = "./Dockerfile", format = ["sdist"] },
+    { path = "./entrypoint.sh", format = ["sdist"] },
+
+    { path = ".examples/crontab", format = ["wheel"] },
+    { path = ".examples/excludarr-example.yml", format = ["wheel"] },
+    { path = ".examples/excludarr.svg", format = ["wheel"] },
+]
+
+[tool.poetry.urls]
+"Bug Reports" = "https://github.com/excludarr/excludarr/issues"
+
+
+[tool.poetry.scripts]
+excludarr = "excludarr.main:cli"
+
+[tool.poetry.dependencies]
+python = ">=3.11,<4.0"
```

Comment:
but is it actually necessary? we never use 3.12 specific features anywhere and it could limit compatibility with some older systems for no reason IMHO. it makes sense to update the docker image but I'm not sure about changing the requirements

Reply:
True, nevermind, the machine I was testing was on the pr #11 by mistake, which had the `type` statement.

All good with this version.